tools/alara_output_processing/alara_output_processing.py

```diff
@@ -3,12 +3,121 @@
 import argparse
 from io import StringIO
 from warnings import warn
+from numpy import array
+from csv import DictReader
+
+# ---------- General Utility Methods ----------
+
+SECONDS_CONV = {'s': 1}
+SECONDS_CONV['m'] = 60  * SECONDS_CONV['s']
+SECONDS_CONV['h'] = 60  * SECONDS_CONV['m']
+SECONDS_CONV['d'] = 24  * SECONDS_CONV['h']
+SECONDS_CONV['w'] = 7   * SECONDS_CONV['d']
+SECONDS_CONV['y'] = 365 * SECONDS_CONV['d']
+SECONDS_CONV['c'] = 100 * SECONDS_CONV['y']
+
+def convert_times(vector, from_unit, to_unit):
+    '''
+    Convert the values in a vector to and from any units in the list of
+        seconds (s), minutes (m), hours (h), days (d), weeks (w), years (y),
+        or centuries (c).
+
+    Arguments:
+        vector (numpy.ndarray): 1-D NumPy array containing time values.
+        from_unit (str): Time units of vector's input state. Accepted values:
+            's', 'm', 'h', 'd', 'w', 'y', 'c'.
+        to_unit (str): Time units of converted vector. Accepted values: 's',
+            'm', 'h', 'd', 'w', 'y', 'c'.
+
+    Returns:
+        converted_vector (numpy.ndarray): 1-D Numpy array of time values
+            converted to the desired units.
+    '''
+
+    return vector * (SECONDS_CONV[from_unit] / SECONDS_CONV[to_unit])
+
+def extract_time_vals(times, to_unit='s'):
+    '''
+    Return the cooling times of an ALARADFrame from their column headers
+        as a list in time units of the user's specification.
+
+    Arguments:
+        times (list): List of cooling times from an ALARA output table header.
+        to_unit (str, optional): Time units for conversion. Accepted values:
+            's', 'm', 'h', 'd', 'w', 'y', 'c'.
+            (Defaults to 's')
+
+    Returns:
+        times (numpy.ndarray): 1-D NumPy array of the converted ALARA cooling
+            times.
+    '''
+    
+    numeric_times = []
+    for time in times:
+        if 'shutdown' in time:
+            numeric_times.append(0.0)
+        else:
+            time_str, unit = time.split('_')
```

Comment:
What if you didn't put the underscore into this during `_normalize_header`?  It seems that you insert the underscore there, and then promptly use it to split this value when you could just use the space?